README.md

```diff
@@ -497,3 +498,17 @@ Some test suites like to segment which tests run based on various facets such as
 ```
 
 You can use this feature in conjunction with Ember Exam's features, which will allow you to segment your test suite but still gain benefits from randomization and splitting.
+
+### Exceeding Browser Timeout
+
+If you have a lot of tests you may run into a timeout error, especially in CI environments with constrained resources.
+
+```
+Error: Browser timeout exceeded: 10s
+```
+
+You can work around this by editing your testem.js to increase `browser_disconnect_timeout` accordingly:
```

Comment:
sorry I forgot about reviewing this PR!

quick question tho -- can you specify where in the testem config this option goes?

maybe specify the `module.exports`?